queries.sql

```diff
@@ -23,3 +23,30 @@ SELECT * FROM animals WHERE name NOT IN ("Gabumon");
 
 -- 8.0 Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)
 SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <=17.3;
+
+-- Part 2 Questions
+
+-- 1. How many animals are there?
+
+SELECT COUNT(*) FROM animals;
+
+-- 2. How many animals have never tried to escape?
+SELECT COUNT(*) FROM animals WHERE escape_attempts=0;
+
+-- 3.0 What is the average weight of animals?
+SELECT AVG(weight_kg)
+FROM animals;
+
```

Comment:
- [ ] It would  be better if we implement the following:

Inside a transaction:

* `DELETE` all animals born after Jan 1st, 2022
* Create a savepoint for the transaction
* `UPDATE` all animals' weight to be their weight multiplied by -1.
* Rollback to the savepoint
* `UPDATE` all animals' weights that are negative to be their weight multiplied by -1.
* Commit transaction